### DISCUSSION TITLE
3 usefull workflow features

### DISCUSSION THREAD

**Discussion by Sogree2** [ransome1/sleek#724] [Ideas]
1) when adding a todo task (using the '+' icon).... press enter to finish/add it (alternative to pressing the "Add" button with the mouse). 
2) Add icon/button to GUI to clear the attribute filters screen (alternative to Ctrl-0)
3) play a nice sound when finishing a todo task...... soooo rewarding (when the user ticks the circle infront of the todo)

**Comment by mshuflin** [comment]
you can use command/control + enter to do #1 on your list

**Comment by cldjr1** [comment]
It seems like the default for creating a todo should be Enter.

**Comment by TangentFoxy** [comment]
Despite personally always entering multiple tasks at a time, where hitting enter to submit a task would break my workflow, I agree that hitting enter should add the task. This is what most people would expect. That it doesn't do this is strange.

Of course, it should either be an option or something that could be ignored with shift+enter.
